docs/mdbook/default.nix

```diff
@@ -398,5 +403,6 @@ pkgs.stdenv.mkDerivation (finalAttrs: {
             echo "- The $link, for use with nixpkgs \`$nixpkgs\`$suffix" >> "$out"
           done
         '';
+    user-configs = callPackage ../user-configs { };
```

Comment:
Hm... Not sure how I feel about naming here.

The page in the docs is "config-examples.md" while the package that's generating them is "user-configs".

Reply:
No hard feeling, both are fine.

Reply:
Ok, I'll leave it as-is then. Having a different name for the actual page in the docs and the packaged used to construct the main part of that page isn't the end of the world.